CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+CHANGELOG
+Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+### Added
+- Github workflow for changelog verification ([#440](https://github.com/opensearch-project/flow-framework/pull/440))
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+
+[Unreleased]: https://github.com/opensearch-project/flow-framework/compare/2.x...HEAD
```

Comment:
This seems like a link to a bunch of commits/diffs which is explicitly what a changelog is NOT supposed to be.  Is this used elsewhere?

Reply:
[OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/CHANGELOG.md?plain=1#L248) has it this way.